### ISSUE TITLE
Not Available.

### ISSUE THREAD

**Issue by D0lcegabbana** [Dr238/SSH-Ramdisk-tool#29]
Hello. iPhone 8 catalina.  
I compiled last version and give this error:

Last login: Mon May  9 13:04:58 on ttys000
cat@iMac-cat ~ % cd /Users/cat/Desktop/SSH-Ramdisk-tool-master\ 2 
cat@iMac-cat SSH-Ramdisk-tool-master 2 % /Users/cat/Desktop/SSH-Ramdisk-tool-master\ 2/CreateRD -d iPhone10,4 -i 14.8 -b D201AP -s /Users/cat/Blobs/2291967019507758_iPhone10\,4_d201ap_15.4.1-19E258_27325c8258be46e69d9ee57fa9a8fbc28b873df434e5e702a8b27999551138ae.shsh2 -p
Not Available.
cat@iMac-cat SSH-Ramdisk-tool-master 2 %

**Comment by Dr238** [comment]
Yeah -p needs some work so I made it not available. For now pwn and remove signature checks with a fork of ipwndfu
